src/components/QueryBuilder.vue

```diff
@@ -173,7 +175,7 @@ export default class QueryBuilder extends Vue {
     text-decoration: underline;
   }
 
-  .fas.fa-question-circle {
+  ::v-deep .fa-question-circle {
```

Comment:
Instead of using the v-deep, maybe you can add a class to the FAIcon component above ?
`<FAIcon class="documentation-help__icon" icon="question-circle" />`
and then `.documentation-help__icon { ...}` in the stylesheet